lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -208,9 +208,14 @@ def test_not_authorized(self):
         self.assertEqual(response.status_code, 404)
 
 
-class CourseListTests(CourseViewTestsMixin, ModuleStoreTestCase):
+class CourseListTests(CourseViewTestsMixin, SharedModuleStoreTestCase):
     view = 'course_structure_api:v0:list'
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Can the common `setUpClass` class method be defined in `CourseViewTestsMixin`?  So it doesn't need to be defined in all the concrete classes?